metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -201,16 +206,21 @@ public ReferenceSequence importReferenceSequence(String accession) {
         Taxonomy taxonomy;
         try {
             // Reference sequences must always be imported through API, even with import mode = DB
-            String referenceSequenceXml = sraxmlRetrieverByAccessionForceApi.getXml(accession);
-            if (referenceSequenceXml.contains(ASSEMBLY_END_TAG)) {
-                AssemblyType assembly = sraAssemblyXmlParser.parseXml(referenceSequenceXml, accession);
-                referenceSequence = referenceSequenceConverter.convert(assembly);
-                taxonomy = taxonomyRepository.findOrSave(extractTaxonomyFromAssembly(assembly));
-            } else if (referenceSequenceXml.contains(ENTRY_END_TAG)) {
-                referenceSequence = sraEntryXmlParser.parseXml(referenceSequenceXml, accession);
+            if (accession.matches("^GCF_.*")) {
```

Comment:
This regex can be made more specific, matching only digits and dots in the second part.